utils/sed/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2018 OpenWrt.org
```

Comment:
Either omit or add copyright to yourself unless you're affiliated with OpenWrt.org